plugins/bitrise-ci/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@internal/plugin-bitrise-ci",
```

Comment:
may need to change this for open sourcing

Reply:
will change the name with the next push